sw-emulator/lib/periph/src/soc_reg.rs

```diff
@@ -804,15 +807,27 @@ impl SocRegistersImpl {
         if size != RvSize::Word {
             Err(BusError::StoreAccessFault)?
         }
+        println!("on_write_flow_status");
```

Comment:
Was this for debug only or we want to keep this?

Reply:
Whoops, thanks for noticing!